### PR TITLE
Added support for encoding in the HttpClient.

### PR DESCRIPTION
**Pull request by pmorelli-origin**
With this change the client will send Accept-Encoding: gzip, deflate; and if the response is compressed the HttpResponseMessage will know how to decompress it for a later deserialization.

### CODE REVIEW

File: hotel-api-sdk/HotelApiClient.cs [hunk 1]
```diff
@@ -223,19 +225,22 @@ public BookingListRS List(List<Tuple<string, string>> param)
...
                         && path.GetType() != typeof(HotelApiPaths.BOOKING_CANCEL) && path.GetType() != typeof(HotelApiPaths.BOOKING_DETAIL) && path.GetType() != typeof(HotelApiPaths.BOOKING_LIST)))
                         throw new Exception("Object request can't be null");
 
                     client.BaseAddress = new Uri(path.getUrl(this.basePath, this.version));
                     client.DefaultRequestHeaders.Clear();
                     client.Timeout = new TimeSpan(0, 0, REST_TEMPLATE_READ_TIME_OUT);
                     client.DefaultRequestHeaders.Add("Api-Key", this.apiKey);
-                    client.DefaultRequestHeaders.Add("Accept-Encoding", "Gzip");
-                    client.DefaultRequestHeaders.TryAddWithoutValidation("Content-Type", "application/json");
-                    client.DefaultRequestHeaders.TryAddWithoutValidation("Accept", "application/json");
```
Comment (alvmunoz) [inline hotel-api-sdk/HotelApiClient.cs]: @pmorelli-origin did you remove these 2 last headers for any specific reason?   I don't think it is related to the compression?

Comment (pmorelli92) [inline hotel-api-sdk/HotelApiClient.cs]: My bad, I was testing something and I forgot to re-add them.

Comment (alvmunoz) [inline hotel-api-sdk/HotelApiClient.cs]: Np, I am readding them.
Thanks.

Comment (pmorelli92) [inline hotel-api-sdk/HotelApiClient.cs]: @alvmunoz can you upload a new nuget version contaning the readding of the mentioned lines?

Comment (alvmunoz) [inline hotel-api-sdk/HotelApiClient.cs]: @pmorelli92 yes, done :)

Comment (pmorelli92) [inline hotel-api-sdk/HotelApiClient.cs]: @alvmunoz thanks!

**Pull request merged**